lib/reporting/reports/suppliers/helpers/columns_helper.rb

```diff
@@ -0,0 +1,121 @@
+# frozen_string_literal: true
+
+module Reporting
+  module Reports
+    module Suppliers
+      module Helpers
+        module ColumnsHelper
+          include LineItemsAccessHelper
+
+          def producer
+            proc { |line_items| supplier(line_items).name }
+          end
+
+          def producer_address
+            proc { |line_items| supplier(line_items).address&.full_address }
+          end
+
+          def producer_abn_acn
+            proc do |line_items|
+              supplier = supplier(line_items)
+
+              supplier.abn.presence || supplier.acn
+            end
+          end
+
+          def email
+            proc { |line_items| supplier(line_items).email_address }
+          end
+
+          def hub
+            proc { |line_items| distributor(line_items).name }
+          end
+
+          def hub_address
+            proc { |line_items| distributor(line_items).address&.full_address }
+          end
+
+          def hub_contact_email
+            proc { |line_items| distributor(line_items).email_address }
+          end
+
+          def order_number
+            proc { |line_items| order(line_items).number }
+          end
+
+          def order_date
+            proc { |line_items| order(line_items).completed_at.strftime("%d/%m/%Y") }
```

Comment:
Do we have a standard way of printing dates? 
Date formats are locale-specific, so this could be confusing (eg in north America they have the day and month the other way around).

Reply:
Hmm, I found a few different ways. But generally they are using a standard format, I guess ISO or compatible. 
Then it's up to the user to load it into their spreadsheet software (eg Excel) and format the date as needed.

https://github.com/openfoodfoundation/openfoodnetwork/blob/master/lib/reporting/reports/orders_and_distributors/base.rb#L10
https://github.com/openfoodfoundation/openfoodnetwork/blob/master/lib/reporting/reports/customers/base.rb#L82
https://github.com/openfoodfoundation/openfoodnetwork/blob/master/lib/reporting/reports/users_and_enterprises/base.rb#L19

`±` I'd suggest `to_date` in this case, which removes the time part, then is implicitly converted `to_s`tring.

Reply:
Yes, that's a good catch to keep in mind of the locale based formatting. Thanks for pointing that out. 
It's fixed here: https://github.com/openfoodfoundation/openfoodnetwork/pull/12879/commits/c8506084e8a7e7a60132a61e22f1e1f11e687736